jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml

```diff
@@ -76,218 +59,104 @@
                 if(gciImageMatcher?.matches()) manager.addShortText("<b>GCI Image: " + gciImageMatcher.group(1) + "</b>", "grey", "white", "0px", "white")
                 def k8sVersionMatcher = manager.getLogMatcher("Using\\spublished\\sversion\\s(.*)\\s\\(from.*")
                 if(k8sVersionMatcher?.matches()) manager.addShortText("<br><b>Kubernetes version: " + k8sVersionMatcher.group(1) + "</b>", "grey", "white", "0px", "white")
+    # Template defaults.
+    jenkins_node: 'e2e'
+    test-owner: 'wonderfly'
+    emails: 'wonderfly@google.com,qzheng@google.com,lakitu-dev@google.com'
 
 - project:
-    name: kubernetes-e2e-gce-gci-ci-master
-    trigger-job: 'kubernetes-build'
-    test-owner: 'wonderfly@google.com'
-    emails: 'wonderfly@google.com,qzheng@google.com'
+    name: kubernetes-e2e-gce-gci-ci
     suffix:
-        # TODO(wonderfly): For GCI, we currently only run CI, slow and serial
-        # tests. More test coverage under way.
-        - 'master':
+        - 'master': # kubernetes-e2e-gce-gci-ci-master
             description: 'Runs all non-slow, non-serial, non-flaky, tests on GCE with GCI images in parallel on the master branch.'
-            timeout: 30
+            timeout: 50
+            trigger-job: 'kubernetes-build'
             job-env: |
-                export JENKINS_GCI_IMAGE_TYPE="dev"
+                # This should become 54 once it's available.
+                export JENKINS_GCI_MILESTONE="53"
```

Comment:
It'll be hard to name it "gci-head" or "gci-canary" as one image can only be in one family. Every image is going to be in some "gci-<milestone>" family already.

On the other hand, I think it makes sense to explicitly pin to a milestone even for k8s `master` branch because ultimately it's their (as a GCI user) choice to make on what milestone they like.